main.go

```diff
@@ -32,6 +32,7 @@ var (
 	libratoUser    = flag.String("user", "", "Librato Username")
 	libratoToken   = flag.String("token", "", "Librato API Token")
 	source         = flag.String("source", "", "Librato Source")
+	url 		   = flag.String("url", "", "Librato Url")
```

Comment:
Can you make `https://metrics-api.librato.com/v1/metrics` the default here instead of empty string? Flags are supposed to be optional if possible.